File: benchmarkDS/Vanilla/CLAUDE_claude-sonnet-4-5-20250929/Lse13.c

#include <stdio.h>

int id_sequence[] = {10, 20, 30, 40, 50, 60, 70, 80, 90, 100};
int id_sequence_size = 10;

int getValueAtIndex(int index) {
    if (index >= 0 && index < id_sequence_size) {
        return id_sequence[index];
    }
    return -1; // Return -1 for invalid index
}

int main() {
    // Test case 1
    int index1 = 0;
    printf("Index %d: %d\\n", index1, getValueAtIndex(index1));
    
    // Test case 2
    int index2 = 5;
    printf("Index %d: %d\\n", index2, getValueAtIndex(index2));
    
    // Test case 3
    int index3 = 9;
    printf("Index %d: %d\\n", index3, getValueAtIndex(index3));
    
    // Test case 4
    int index4 = -1;
    printf("Index %d: %d\\n", index4, getValueAtIndex(index4));
    
    // Test case 5
    int index5 = 10;
    printf("Index %d: %d\\n", index5, getValueAtIndex(index5));
    
    return 0;
}

File: benchmarkDS/Vanilla/GEMINI_gemini-2.5-pro/Lse13.java
public class Lse13 {
    // A predefined sequence of IDs
    private static final int[] id_sequence = {101, 102, 103, 104, 105, 106, 107, 108, 109, 110};

    /**
     * Returns the value at the given index in the id_sequence.
     *
     * @param index The index of the value to retrieve.
     * @return The value at the specified index, or -1 if the index is out of bounds.
     */
    public static int getValueAtIndex(int index) {
        if (index >= 0 && index < id_sequence.length) {
            return id_sequence[index];
        } else {
            return -1; // Return -1 to indicate an invalid index
        }
    }

    public static void main(String[] args) {
        // Test Case 1: Valid index at the beginning
        int index1 = 0;
        System.out.println("Value at index " + index1 + ": " + getValueAtIndex(index1));

        // Test Case 2: Valid index in the middle
        int index2 = 4;
        System.out.println("Value at index " + index2 + ": " + getValueAtIndex(index2));

        // Test Case 3: Valid index at the end
        int index3 = 9;
        System.out.println("Value at index " + index3 + ": " + getValueAtIndex(index3));

        // Test Case 4: Invalid index (out of bounds high)
        int index4 = 10;
        System.out.println("Value at index " + index4 + ": " + getValueAtIndex(index4));

        // Test Case 5: Invalid index (negative)
        int index5 = -1;
        System.out.println("Value at index " + index5 + ": " + getValueAtIndex(index5));
    }
}